routes/theatre.routes.js

```diff
@@ -1,14 +1,19 @@
 const theatreController = require('../controllers/theatre.controller')
-const {authJwt} = require('../middlewares')
+const {authJwt, paramsValidator, validaionsOfTheatreReqBody} = require('../middlewares')
 
  
 module.exports = (app)=>{
-    app.post("/mba/api/v1/theatres", [authJwt.verifyToken, authJwt.isTheatreOwnerOrAdmin], theatreController.createNewTheatre);
-    app.put("/mba/api/v1/theatres/:id", [authJwt.verifyToken, authJwt.isTheatreOwnerOrAdmin, authJwt.isValidTheatreOwner], theatreController.editTheatre)
-    app.delete("/mba/api/v1/theatres/:id", [authJwt.verifyToken, authJwt.isTheatreOwnerOrAdmin, authJwt.isValidTheatreOwner], theatreController.deleteTheatre)
+    app.post("/mba/api/v1/theatres", [authJwt.verifyToken, authJwt.isTheatreOwnerOrAdmin, validaionsOfTheatreReqBody.validateTheatreBody], theatreController.createNewTheatre);
+
+    app.put("/mba/api/v1/theatres/:id", [authJwt.verifyToken,  paramsValidator.theatreParams, authJwt.isTheatreOwnerOrAdmin], theatreController.editTheatre)
```

Comment:
Same comment as before...try to add validation for the request body as well

Reply:
You can like to re-order the middlewares... check for Admin/Owner before any other validation